chart/templates/manager-rbac.yaml

```
@@ -212,6 +212,32 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - secrets.hashicorp.com
+  resources:
+  - hcpauths/finalizers
+  verbs:
+  - update
+- apiGroups:
+  - secrets.hashicorp.com
+  resources:
+  - hcpauths/status
+  verbs:
+  - get
+  - patch
+  - update
+- apiGroups:
+  - secrets.hashicorp.com
+  resources:
+  - vaultauths
```

This probably needs to be hcpauths?

```suggestion
  - hcpauths
```

---

yup, bad merge. fixing..

---

Fixed with 510c96aebdc5590f24654c91c9e5caaecc0d8647